### PR TITLE
non-split-orders-endpoint-tests

### PR DESCRIPTION
**Pull request by shubham-khandelwal2-metacube**
## PR Description

https://spryker.atlassian.net/browse/CC-16823

### CODE REVIEW

**Review by Katerynaa-Lavryk — DISMISSED** [review]

File: tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot [hunk 1]
```diff
@@ -0,0 +1,493 @@
+*** Settings ***
+Suite Setup    SuiteSetup
+Test Setup    TestSetup
+Resource    ../../../../../../resources/common/common_api.robot
+Default Tags    glue
+
+*** Test Cases ***
+ENABLER
+    TestSetup
+#GET requests
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot:10]: please add a test for getting order that has configurable bundle

File: tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot [hunk 2]
```diff
@@ -0,0 +1,493 @@
...
+
+#fails CC-19233
+Get_order_by_order_id
```
Comment (OKrivtsova) [inline tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot:13]: please also add a test for order with gift card

File: tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot (hunk 1 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot:10]: DONE

File: tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot (hunk 2 above)
Comment (shubham-khandelwal2-metacube) [reply tests/api/suite_nonsplit/glue/order_endpoints/orders/positive.robot:12]: DONE

**Review by OKrivtsova — DISMISSED** [review]

**Review by OKrivtsova — APPROVED** [review]

**Pull request merged**